tendrl/commons/fixtures/nodecontext.py

```diff
@@ -0,0 +1,4 @@
+class NodeContext():
+
+    def __init__(self,*args):
+        NodeContext = "Test Code"
```

Comment:
use actual node_id, machine_id and all other attributes (assign them fake values) from the original NodeContext , TendrlContext classes